Pipfile

```diff
@@ -8,16 +8,15 @@ verify_ssl = true
 
 [packages]
 boto3 = "*"
-aws-wsgi = {ref = "a5b800373c326e75f705d400c48f7df301409724", git = "https://github.com/DemocracyClub/awsgi.git"}
+aws-wsgi = {git = "https://github.com/DemocracyClub/awsgi.git"}
```

Comment:
There are 3 VCS dependencies in this file
- aws-wsgi
- dc_design_system
- django-uk-political-parties
I'm aware that they're pinned to a commit hash in Pipfile.lock but I think it would make it easier to understand what version we are _intending_ to be on if we pin all 3 to a ref (tag or commit) in Pipfile too.

Reply:
Fixed with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/1bb47cdcc47a4138d736b166d4e51eb08a2372b5. I haven't updated the `deploy-requirements.txt` file until you confirm what's needed there.